vita3k/modules/SceAudio/SceAudio.cpp

```diff
@@ -91,9 +91,19 @@ EXPORT(int, sceAudioOutOpenPort, SceAudioOutPortType type, int len, int freq, Sc
     port->shared.stream = stream;
 
     const std::lock_guard<std::mutex> lock(host.audio.shared.mutex);
-    const int port_id = host.audio.shared.next_port_id++;
+
+    const int port_id = host.audio.shared.next_port_id;
     host.audio.shared.out_ports.emplace(port_id, port);
 
+    //find next lowest available port
+    int highest_port_id = host.audio.shared.out_ports.rbegin()->first;
+    for (int i = 1; i < highest_port_id + 1; i++) {
+        if (!host.audio.shared.out_ports.find(i)->second) {
+            host.audio.shared.next_port_id = i;
+            return i;
+        }
+    }
+    host.audio.shared.next_port_id = port_id + 1;
```

Comment:
```suggestion
    host.audio.shared.next_port_id = port_id++;
```

Reply:
This throws an error for me when I tried it

Reply:
oh, maybe ++port_id

Reply:
ahnoà sorry, is const lol

Reply:
"expression must be modifiable lvalue"

Reply:
pour les deux

Reply:
tes, i know no worry ^^